pkg/features/list.go

```diff
@@ -102,4 +102,7 @@ var (
 
 	// CloudSources enables support for cloud source integrations.
 	CloudSources = registerFeature("Enable support for cloud source integrations", "ROX_CLOUD_SOURCES", true)
+
+	// ComplianceHierachyControlData enables support for compliance hierarchy control data.
+	ComplianceHierachyControlData = registerFeature("Enable support to display and import the compliance hierarchy control data", "ROX_COMPLIANCE_HIERARCHY_CONTROL_DATA", true)
```

Comment:
we should probably start with it as `false`

Reply:
move fast and break things, no ? :D

Reply:
Fixed it